Validation/RecoEgamma/plugins/ElectronMcMiniAODSignalValidator.h

```diff
@@ -17,6 +17,8 @@
 
 #include "DataFormats/VertexReco/interface/VertexFwd.h"
 #include "DataFormats/VertexReco/interface/Vertex.h"
+#include "DataFormats/Candidate/interface/Candidate.h"
```

Comment:
there is no need of it. It was include when I added the recomp modifications from another file by copy-paste. Do you want I remove it and resub ?